pytest_mpi/plugin.py

```diff
@@ -93,10 +93,6 @@ def pytest_collection_modifyitems(config, items):
 
     _plugin_in_use = any(item.get_closest_marker("parallel") for item in items)
 
-    # if mpi-pytest is not being used then don't mark anything
-    if not _plugin_in_use:
-        return
-
```

Comment:
Why change this?

Reply:
Answer from offline: this messes things up if none of the _selected_ tests have a parallel mark, but there are tests with a parallel mark in the full suite.